src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -173,7 +173,8 @@ public PlantGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 
 		onBlockAdded = new ProcedureSelector(this.withEntry("block/when_added"), mcreator,
 				L10N.t("elementgui.plant.event_on_added"),
-				Dependency.fromString("x:number/y:number/z:number/world:world/blockstate:blockstate"));
+				Dependency.fromString(
+						"x:number/y:number/z:number/world:world/blockstate:blockstate/oldState:blockstate/moving:logic"));

```

Comment:
Consider renaming oldState to oldstate as so far afaik all dependencies are lowercase only, same with hitX, ...

Reply:
Particles have a `onGround` dependency

Reply:
Hmm, that is correct, but in most places they are lowercase. Decide yourself ;)

Reply:
I think it's fine to keep it as is. Imo camelCase makes them more readable